src/main/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSink.java

```diff
@@ -83,24 +88,58 @@ public SparkHeaderlessBAMOutputFormat() {
         }
     }
 
+    // Output format class for writing CRAM files through saveAsNewAPIHadoopFile. Must be public.
+    public static class SparkCRAMOutputFormat extends KeyIgnoringCRAMOutputFormat<NullWritable> {
```

Comment:
This class is identical to `SparkBAMOutputFormat` -- is it only a separate class because `setHeader()` is static and we might want a different header for the two output formats, or is there some deeper reason?